src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import Sidebar from './Sidebar';
+import styles from './App.css';
+
+export default function App() {
+  const [selected, updateSelected] = useState('home');
+
+  const content = {
+    home: 'You have made it',
+    about: 'You are reading about me',
+    contact: 'do not call me',
+    projects: 'none yet',
+  };
+
+  return (
+    <div className={styles.App}>
+      <Sidebar select={updateSelected}>
```

Comment:
I don't believe you need the prop ```select={updateSelected}``` since it's being passed down with the children.